service/captcha/capthca.go

```diff
@@ -0,0 +1,97 @@
+package captcha
```

Comment:
There's a typo in these filenames:

- templates/_capthca.html
- service/captcha/capthca.go